Provider/FileProvider.php

```diff
@@ -401,7 +401,10 @@ protected function doTransform(MediaInterface $media)
             $media->setProviderReference($this->generateReferenceName($media));
         }
 
-        if ($media->getBinaryContent() instanceof File) {
+        if ($media->getBinaryContent() instanceof UploadedFile) {
+            $media->setContentType($media->getBinaryContent()->getClientMimeType());
```

Comment:
Can the client mimetype be null?

Reply:
`UploadedFile` defaults to a mimetype of `application/octet-stream` should the mimetype constructor argument be null.

https://github.com/symfony/http-foundation/blob/master/File/UploadedFile.php#L91